CONTRIBUTING.md

```diff
@@ -436,12 +436,21 @@ Sometimes signatures will point directly to the issue (eg, dispatch), though som
 - Merge commits are not allowed in our main branch. We enforce a linear
   history.
 
+- You can use either of the following methods to merge your branch on the
+  GitHub UI:
+  - Squash and merge
+  - Rebase and merge
+
+  If you use squashing, when GitHub asks you to enter a new commit message,
+  ensure that your commit message follows our required format as outlined above
+  in this section. Failure to do so is a violation of our standards.
```

Comment:
@tt-rkim , I never saw github asking. My expectation is that it uses PR Title for a new commit and PR description for an extended commit description.

Reply:
<img width="952" alt="Screenshot 2024-05-14 at 12 48 38 PM" src="https://github.com/tenstorrent/tt-metal/assets/109366641/2c2feb09-8dc1-43fa-afdc-c2ecdb10721f">
These fields are editable once you click Squash and Merge
Do you not see that?

Reply:
Oh, thats right. Thank you!